lang/es-es.js

```diff
@@ -7,6 +7,12 @@ export default {
 	"components.calendar.selected": "Seleccionado.",
 	"components.calendar.show": "Mostrar {month}",
 	"components.dialog.close": "Cerrar este cuadro de diálogo",
+	"components.filter.loading": "Loading filters",
+	"components.filter.filterCount": "{number, select, max {99+} all {All} other {{números}}}",
```

Comment:
"All" didn't get translated here but it did for `es` so I used that one. Also they translated `{number}` which I fixed.

```suggestion
	"components.filter.filterCount": "{number, select, max {99+} all {Todos} other {{number}}}",
```